src/shell/components/InstanceFlyoutMenuModal/DomainsMenu.tsx

```diff
@@ -0,0 +1,107 @@
+import { useSelector } from "react-redux";
+import {
+  Box,
+  Button,
+  Dialog,
+  DialogContent,
+  DialogTitle,
+  IconButton,
+  Typography,
+  ListItemButton,
+  ListItemIcon,
+  ListItemText,
+  ListItem,
+} from "@mui/material";
+import ArrowBackIcon from "@mui/icons-material/ArrowBack";
+import LanguageIcon from "@mui/icons-material/Language";
+import RemoveRedEyeIcon from "@mui/icons-material/RemoveRedEye";
+import OpenInNewIcon from "@mui/icons-material/OpenInNew";
+interface Props {
+  onClose?: () => void;
+  instanceZUID?: string;
+}
+
+const DomainsMenu = ({ onClose, instanceZUID }: Props) => {
+  // @ts-ignore
+  // @Note: check if we can migrate this to rtk
+  const instance = useSelector((state) => state.instance);
+
+  return (
+    <Dialog
+      PaperProps={{
+        style: {
+          position: "absolute",
+          bottom: 0,
+          left: 0,
+          width: "448px",
+          height: "392px",
+        },
+      }}
+      open={true}
+      onClose={onClose}
+      fullWidth
+      maxWidth={"xs"}
+    >
+      <Box sx={{ display: "flex", justifyContent: "space-between", p: 2 }}>
+        <ListItem sx={{ p: 0, width: 0 }}>
+          <ListItemIcon sx={{ minWidth: "30px" }} onClick={() => onClose()}>
+            <IconButton sx={{ p: 0 }}>
```

Comment:
If I don't remove the padding it will look like this:
<img width="619" alt="Screenshot 2022-12-31 at 10 17 50 PM" src="https://user-images.githubusercontent.com/40680430/210139786-a637d6eb-2831-40df-89ca-d884fbba59ec.png">

Reply:
You're right. My reason for using that is just so I could use some default list styles. I don't wanna have a box and customize all the styling in that box. But maybe that's better.